enable/scrolled.py

```diff
@@ -612,7 +612,7 @@ def _container_handle_mouse_event(self, event, suffix):
     # -------------------------------------------------------------------------
 
     def __getstate__(self):
-        state = super(Scrolled, self).__getstate__()
+        state = super().__getstate__()
         for key in ["alternate_vsb", "_vsb", "_hsb"]:
             if key in state:
                 del state[key]
```

Comment:
orthogonal but these could be updated to use `transient` metadata

Reply:
i'll open a separate issue/PR for this.

Reply:
i created https://github.com/enthought/enable/issues/795